src/riak_core_gen_server.erl

```diff
@@ -1015,8 +1015,8 @@ format_status(Opt, StatusData) ->
 		      Name
 	      end,
     Header = lists:concat(["Status for generic server ", NameTag]),
-    Log = sys:get_debug(log, Debug, []),
-    Specfic = 
+    Log = sys:get_log(Debug),
```

Comment:
src/riak_core_gen_server.erl
1018: Call to missing or unexported function sys:get_log/1

Reply:
Dialyzer doesn't like this in OTP 20